packages/mobx/src/api/extendobservable.ts

```diff
@@ -41,6 +44,8 @@ export function extendObservable<A extends Object, B extends Object>(
     const descriptors = getOwnPropertyDescriptors(properties)
 
     const adm: ObservableObjectAdministration = asObservableObject(target, options)[$mobx]
+    const allowStateChanges = allowStateChangesStart(true)
+    globalState.suppressReportChanged = true
```

Comment:
The plan is to change these into `suppressReportChangedStart/End` and probably move it to `initObservableStart/End` (together with `unstrackedStart/End` and `allowStateChangesStart/End` as noted in comments).

Reply:
yeah, some dedicated utility sounds great here to avoid copy / paste mistakes.